paper/tsd2017.tex

```diff
@@ -40,11 +40,11 @@
 
 % TSD 2017: Put your title here (please, use capitalization, see e.g.
 % http://en.wikibooks.org/wiki/Basic_Book_Design/Capitalizing_Words_in_Titles)
-\title{Language-Independent Acoustic Modeling for Medieval Latin}
+\title{Language Specific Knowledge and Data Free Acoustic Modeling for Medieval Latin LVCSR}
```

Comment:
Néhány észrevételem a címmel kapcsolatban:

+ a 'data free' olyan mintha adat nélkül csinálnánk felismerést
+ a 'language-independent' kifejezésnek szakirodalmi beágyazottsága van , és pont azt jelenti, amit csinálunk:
    + más nyelvekből veszünk kölcsön adatot egy bizonyos under-resourced nyelv akusztikus modellezéséhez
    + ld. a Besacier és T. Schultz cikkeket
+ ha a 'language specific knowledge free' kifejezést a címben akarjuk tartani, akkor nagy mértékben át kell írni a cikket, mert a mostani felépítés szerint kétféle irányban fejlesztjük az akusztikus/kiejtésmodellezést:
    + *knowledge*-based g2p-vel
    + és a valóban knowledge-free USG módszerrel
    + de akkor a cím csak a cikk felére igaz, úgyhogy vagy ki kéne szedni a g2p-s részeket, vagy valahogyan átírni a cikket, hogy sokkal több hangsúly essen az USG-re
+ az a javaslatom, hogy
     + egy külön pull request-ben a fent leírt változtatásokkal változtassuk meg a címet, vagy
     + tartsuk meg a mostanit